pkg/asset/internal/templates.go

```diff
@@ -419,4 +420,42 @@ spec:
     port: 53
     protocol: TCP
 `)
+	EtcdOperatorTemplate = []byte(`apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: etcd-operator
+  namespace: kube-system
+  labels:
+    k8s-app: etcd-operator
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        k8s-app: etcd-operator
+    spec:
+      containers:
+      - name: etcd-operator
+        image: quay.io/coreos/etcd-operator
+        env:
+        - name: MY_POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+`)
+	EtcdSvcTemplate = []byte(`apiVersion: v1
+kind: Service
+metadata:
+  name: etcd-service
+  namespace: kube-system
+spec:
+  selector:
+    app: etcd
+    etcd_cluster: etcd-cluster
+  clusterIP: 10.3.0.20
```

Comment:
per @dghubble comment, lets start with a different default so we're not conflicting from day 1. maybe `.15`? Any objections?